test/fixtures/add/component.yaml

```diff
@@ -0,0 +1,11 @@
+name: add
```

Comment:
The `TestAdd` test actually writes this component, so we don't need it checked in.